3.Tests/UI/Hooks/ProjectHooks.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using RestSharp;
+using TechTalk.SpecFlow;
+using Todoly.Core.Helpers;
+
+namespace SeleniumTest.Tests.Hooks;
+
+[Binding]
+public class ProjectHooks
+{
+    private readonly ScenarioContext _scenarioContext;
+    private readonly RestHelper _client;
+    private readonly string _url;
+    private readonly string _projectName;
+
+    public ProjectHooks(ScenarioContext scenarioContext)
+    {
+        _scenarioContext = scenarioContext;
+        _client = new RestHelper(ConfigModel.ApiHostUrl);
+        _url = ConfigModel.ApiProjectUri;
+        _projectName = IdHelper.GetNewId();
+    }
+
+    [BeforeScenario("create.project")]
+    public void CreateProject()
+    {
+        string payload = $"{{ \"Content\": \"{_projectName}\" }}";
+
+        _client.Authenticate(ConfigModel.TODO_LY_EMAIL, ConfigModel.TODO_LY_PASS);
+
+        _scenarioContext["Response"] = _client.Post(_url, payload);
```

Comment:
No need to store the whole API response on the context, you should store the ProjectModel directly

Reply:
Should we store the projectmodel in the Scenario?

Reply:
Not really, only if you need to, I was just saying that it would be better to just store the ProjectModel from the response body instead of the RestResponse object

Reply:
It's done, thank you.